components/pricing/PricingTable.js

```diff
@@ -74,6 +74,18 @@ const StyledTable = styled(Box)`
     border-bottom-right-radius: 8px;
   }
 
+  @media screen and (max-width: 62em)
```

Comment:
@AARYAN-MAHENDRA : If you look closely at the [failing pull request](https://github.com/opencollective/opencollective-frontend/pull/4586/checks?check_run_id=835177620) you will see that it says, 

```
Checking formatting...
components/pricing/PricingTable.js
Code style issues found in the above file(s). Forgot to run Prettier?
```

This means that there's a formatting problem in the file `PricingTable.js`. This repo uses [prettier](https://prettier.io/) to check for correct formatting. All you got to do is run `npm run prettier:write` on your branch and it will fix it for you. This is probably coming from this code block where the alignment is off. :smile:

Reply:
oh, ok thanks a lot, @SudharakaP

Reply:
> oh, ok thanks a lot, @SudharakaP

you are welcome. :+1: